src/cascadia/TerminalSettings/TerminalSettings.cpp

```diff
@@ -3,81 +3,11 @@
 
 #include "pch.h"
 #include "TerminalSettings.h"
-#include <DefaultSettings.h>
 
 #include "TerminalSettings.g.cpp"
 
 namespace winrt::Microsoft::Terminal::Settings::implementation
 {
-    // Disable "default constructor may not throw."
-    // We put default values into the hstrings here, which allocates and could throw.
-    // Working around that situation is more headache than it's worth at the moment.
-#pragma warning(suppress : 26455)
-    TerminalSettings::TerminalSettings() :
-        _defaultForeground{ DEFAULT_FOREGROUND_WITH_ALPHA },
-        _defaultBackground{ DEFAULT_BACKGROUND_WITH_ALPHA },
-        _selectionBackground{ DEFAULT_FOREGROUND },
-        _colorTable{},
-        _historySize{ DEFAULT_HISTORY_SIZE },
-        _initialRows{ 30 },
-        _initialCols{ 80 },
-        _rowsToScroll{ 0 },
-        _snapOnInput{ true },
-        _cursorColor{ DEFAULT_CURSOR_COLOR },
-        _cursorShape{ CursorStyle::Vintage },
-        _cursorHeight{ DEFAULT_CURSOR_HEIGHT },
-        _wordDelimiters{ DEFAULT_WORD_DELIMITERS },
-        _copyOnSelect{ false },
-        _profileName{},
-        _useAcrylic{ false },
-        _tintOpacity{ 0.5 },
-        _padding{ DEFAULT_PADDING },
-        _fontFace{ DEFAULT_FONT_FACE },
-        _fontSize{ DEFAULT_FONT_SIZE },
-        _fontWeight{ DEFAULT_FONT_WEIGHT },
-        _backgroundImage{},
-        _backgroundImageOpacity{ 1.0 },
-        _backgroundImageStretchMode{ winrt::Windows::UI::Xaml::Media::Stretch::UniformToFill },
-        _backgroundImageHorizontalAlignment{ winrt::Windows::UI::Xaml::HorizontalAlignment::Center },
-        _backgroundImageVerticalAlignment{ winrt::Windows::UI::Xaml::VerticalAlignment::Center },
-        _keyBindings{ nullptr },
-        _scrollbarState{ ScrollbarState::Visible },
-        _antialiasingMode{ TextAntialiasingMode::Grayscale },
-        _forceFullRepaintRendering{ false },
-        _softwareRendering{ false }
-    {
-    }

```

Comment:
Wait...don't we still need this?

Reply:
Nope! The default values are passed directly in the `GETSET_PROPERTY` statement now ☺️